### ISSUE TITLE
Support bootswatch themes?

### ISSUE THREAD

**Issue by Fabryprog** [angular-ui/ui-grid#2020]
Hello,

i included an bootstrap  theme from bootswatch project (https://github.com/thomaspark/bootswatch/)

unluckily ui-grid had same theme.

Can i set similar (or same) theme to ng-grid component?

**Comment by shenlong** [comment]
hi @tepez I saw your gist. Seems like we need to re-compile the grid-ui.
Is there a CSS overwrite to achieve similar effects?

**Comment by tepez** [comment]
I doubt there is a CSS like this publicly available because someone would
have to maintain it and re-compile the less files every time ui-grid is
updated.

**Comment by ghiscoding** [comment]
You might want to take a look at the [forked plunker](http://plnkr.co/nZEGqK), I forked from @qorsmond. It's just 1 CSS file only, there is no need to recompile or anything, just run it. I got many element working and using Bootstrap. I also use some Glyphicons of Bootstrap to overwrite some of the grid icons.

Anyway take a look at the [open issue 2586](https://github.com/angular-ui/ui-grid/issues/2586), I posted some print screens near the bottom (or run the [plunker](http://plnkr.co/nZEGqK))

I wish someone could push the code on the official UI Grid.
